.neutrinorc.js

```diff
@@ -79,6 +168,10 @@ module.exports = {
           bindings: 'bindings'
         }));
 
+      // TODO: Why is this needed?
+      neutrino.config.resolve.modules.add(MODULES);
+      // neutrino.config.resolveLoader.modules.add(MODULES);
+
       neutrino.config.output.publicPath('/');
```

Comment:
This can be set via the `@neutrinojs/react` options now, which will also ensure the devserver equivalent option is set to a matching value automatically.